xlators/cluster/dht/src/dht-hashfn.c

```diff
@@ -92,13 +90,13 @@ dht_hash_compute(xlator_t *this, int type, const char *name, uint32_t *hash_p)
         }
 
         if (!munged && priv->rsync_regex_valid) {
-            gf_msg_trace(this->name, 0, "trying regex for %s", name);
             munged = dht_munge_name(name, rsync_friendly_name, len,
                                     &priv->rsync_regex);
         }
     }
     UNLOCK(&priv->lock);
-    if (munged) {
+
+    if (caa_unlikely(munged)) {
```

Comment:
I would also remove `caa_unlikely()` from here. It depends on the workload and could fail constantly.